packages/forms/src/UIForm/fieldsets/CollapsibleFieldset/CollapsibleFieldset.component.js

```diff
@@ -0,0 +1,94 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '@talend/react-components/lib/Icon';
+import classNames from 'classnames';
+import Widget from '../../Widget';
+
+import theme from './CollapsibleFieldset.scss';
+
+function defaultTitle(_, schema) {
```

Comment:
Why the underscore?

Reply:
It's to say `whatever the name is, I don't use it in this implem`

Reply:
Then why is the argument here?

Reply:
It's the value.
* the value is the 1st arg for retrocompat
* In arrays, each value is the item, so you can create a title depending on each item (you need it in your project)

Reply:
OK. I just hope it doesn't become a trend :wink: